huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_kafka_consumer_group_test.go

```diff
@@ -0,0 +1,139 @@
+package dms
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getDmsKafkaConsumerGroupResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
+	// getKafkaConsumerGroup: query DMS kafka consumer group
+	var (
+		getKafkaConsumerGroupHttpUrl = "v2/{project_id}/instances/{instance_id}/groups/{group}"
+		getKafkaConsumerGroupProduct = "dmsv2"
+	)
+	getKafkaConsumerGroupClient, err := cfg.NewServiceClient(getKafkaConsumerGroupProduct, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating DmsKafkaConsumerGroup Client: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error creating DMS Client: %s", err)
```
and fix other places

Reply:
done